index.html

```diff
@@ -39,6 +39,8 @@
         <!--TODO move these script to its own javascipt intilisation file -->
         <script src="https://npmcdn.com/leaflet@1.0.0-rc.3/dist/leaflet.js"></script>
         <script type="text/javascript" src="js/irish-counties-data.js"></script>
+        <!-- Only Load this when dublin is zoomed in -->
+        <script type="text/javascript" src="js/dublinPrimarySchools.js"></script>
```

Comment:
Got it from department education...but I had to geocode them myself with some web app that's why there only a few Dublin ones done for POC